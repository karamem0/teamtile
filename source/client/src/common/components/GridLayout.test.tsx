//
// Copyright (c) 2021-2025 karamem0
//
// This software is released under the MIT License.
//
// https://github.com/karamem0/teamtile/blob/main/LICENSE
//

import React from 'react';

import { expect, it } from 'vitest';
import { render, screen } from '@testing-library/react';
import Presenter from './GridLayout.presenter';

it('should create a shapshot', () => {
  const params = {
    children: (
      <div data-testid="test-Children" />
    )
  };
  const { asFragment } = render(
    <Presenter {...params} />
  );
  expect(asFragment()).toMatchSnapshot();
  expect(screen.getByTestId('test-Children')).toBeInTheDocument();
});
