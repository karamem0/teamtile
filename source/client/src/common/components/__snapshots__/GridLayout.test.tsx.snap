// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create shapshot 1`] = `
.emotion-0 {
  display: grid;
  grid-template-rows: auto;
  grid-template-columns: repeat(auto-fill, minmax(22.5rem, 1fr));
  grid-gap: 0.5rem;
}

@media (width >= 600px) {
  .emotion-0 {
    grid-gap: 1rem;
  }
}

<body>
  <div>
    <div
      class="emotion-0"
      role="list"
    />
  </div>
</body>
`;
